reference/stories/functions.lasagna-master.md

```diff
@@ -0,0 +1,38 @@
+# Lasagna Recipe
+
+## Story
+
+This exercise builds on the original lasagna story to expand on the creation of basic [functions][types-function], and add in [default arguments][concepts-default_arguments], [named parameters][concepts-named_parameters], [variadic functions][concepts-variadic_functions], [multiple return values][concepts-multiple_return_values], [inout (reference) parameters][concepts-inout_parameters], and [nested functions][concepts-nested_functions].
```

Comment:
This section should contain the first paragraph of the story as found in the `instructions.md` file. Could you copy-paste that here?